cog/class_role.py

```
@@ -12,7 +12,9 @@
 
 def get_courses():
     try:
-        with open(f"{os.getcwd()}/DataBase/clas.json", "r", encoding="utf-8") as file:
+        with open(
+            f"{os.getcwd()}/database/courses.json", "r", encoding="utf-8"
```

Ruff (0.14.7)

Unnecessary mode argument

Remove mode argument

(UP015)